src/khoj/interface/web/config.html

```diff
@@ -122,13 +158,39 @@ <h3 class="card-title">API Keys</h3>
             </div>
         </div>
     </div>
-    <div class="section">
-        <h2 class="section-title">Manage Data</h2>
-        <div class="section-manage-files">
-            <div id="delete-all-files" class="delete-all=files">
-                <button id="delete-all-files" type="submit" title="Delete all indexed files">🗑️ Remove All</button>
-            </div>
-            <div class="indexed-files">
+     <div class="section">
+        <h2 class="section-title">Billing</h2>
+        <div class="section-cards">
+            <div class="card">
+                <div class="card-title-row">
+                    <img class="card-icon" src="/static/assets/icons/credit-card.png" alt="Credit Card">
+                    <h3 class="card-title">
+                        <span>Subscription</span>
+                        <img id="configured-icon-subscription"
+                            style="display: {% if not is_subscribed %}none{% endif %}"
+                            class="configured-icon"
+                            src="/static/assets/icons/confirm-icon.svg"
+                            alt="Configured">
+                    </h3>
+                </div>
+                <div class="card-description-row">
+                    <p class="card-description">Manage your subscription to Khoj Cloud</p>
+                </div>
+                {% if not is_subscribed %}
+                <div class="card-action-row">
+                    <a class="card-button happy" href="https://buy.stripe.com/28ocQb7kb4iD0cEdQQ?email={{ username }}" target="_blank">
```

Comment:
nit: maybe use `happy` for after subscription is active?

Also, when I click on the link, it doesn't autopopulate my email address in the checkout page.

Reply:
And when the `stripe` settings are not configured, this card shouldn't appear.

Reply:
Should we drive the link here through an environment variable and pass it to the template so we can update it dynamically when required?

Reply:
- Fixed auto-population for my email address
- Hid the billing card if strip environment not setup
- Now using environment variable `KHOJ_CLOUD_SUBSCRIPTION_URL` to set the Stripe payment link
- Letting the subscribe card be happy colored for now :)